antora/docs/modules/ROOT/templates/_rules.hbs

```diff
@@ -24,6 +24,13 @@
 * Code: `{{ packageInfo.shortName }}.{{ shortName }}`
 {{#if effectiveOn}}
 * Effective from: `{{ effectiveOn }}`{{/if}}
-* https://github.com/enterprise-contract/ec-policies/blob/main/{{ file }}#L{{ row }}[Source, window="_blank"]
+{{#if packageInfo.shortName}}
+    {{#if (isInArray packageInfo.shortName '["builtin_attestation", "builtin_image"]')}}
```

Comment:
We could future proof this a bit, and do something like:

```suggestion
    {{#if isBuiltin packageInfo.shortName }}
```

Reply:
I'd originally taken this approach but upon further consideration decided to do a more generic function in case we need to evaluate array membership in other situations.

Reply:
With this any new builtin check would need a change here, which for me is a big drawback of this approach.
The implementation of `isBuiltin` could look at the `builtin_` prefix and having it in the template as `isBuiltin` makes the template clearer. We currently don't need a `is*` function, so when such requirement occurs we can refactor to something like this`.

Reply:
I appreciate the feedback. We currently only have a single place (here) where we need to check array membership, but this approach is able handle any use case whereas the `isBuiltIn` approach would only evaluate a single use case. I'm going to proceed with this implementation. Feel free to follow-up if you feel it's worthwhile.